### ISSUE TITLE
How to config parameter's order of signature?

### ISSUE THREAD

**Issue by erhu** [sourcegraph/issues#121]
Hello, does the sourcegraph have a configuration that modifies the order of the parameters of the build signature? We found that the order used by the sourchgraph is RelayState + SAMLRequest + SigAlg. But the order given in the official documentation is: SAMLRequest + RelayState + SigAlg. 
See https://www.oasis-open.org/committees/download.php/56779/sstc-saml-bindings-errata-2.0-wd-06.pdf
![image](https://user-images.githubusercontent.com/996319/44127798-99cb346a-a071-11e8-95d6-5e4650b9777a.png)

**Comment by beyang** [comment]
Thanks for reporting! We've identified the source of this issue (https://github.com/russellhaering/gosaml2/issues/44) and are working on a fix.

**Comment by beyang** [comment]
@erhu This should now be fixed in Sourcegraph Server `2.10.3`, which you can run via the following command:
```
docker run --publish 7080:7080 --rm --volume ~/.sourcegraph/config:/etc/sourcegraph --volume ~/.sourcegraph/data:/var/opt/sourcegraph --volume /var/run/docker.sock:/var/run/docker.sock sourcegraph/server:2.10.3
```

Thanks for using Sourcegraph and please let us know if this issue still occurs.
